engine/graphics/src/graphics.cpp

```diff
@@ -189,6 +188,37 @@ namespace dmGraphics
         return 0x0;
     }
 
+    HVertexStreamDeclaration NewVertexStreamDeclaration(HContext context)
```

Comment:
context isn't used but we don't know if that will change in the future so might as well pass it in here